templates/components/products/price.html

```diff
@@ -1,15 +1,20 @@
+<!--products have options-->
+{{#if has_options}}
 {{#if price.with_tax}}
     <div class="price-section price-section--withTax" {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
         {{#if price.rrp_with_tax}}
             <span data-product-rrp-with-tax class="price price--rrp">{{price.rrp_with_tax.formatted}}</span>
         {{/if}}
+        {{#if theme_settings.price_as_low_as}}
+        <span data-product-price-with-tax class="price">As low as {{price.with_tax.formatted}}</span>
+        {{else}}
         <span data-product-price-with-tax class="price">{{price.with_tax.formatted}}</span>
+        {{/if}}
         {{#if price.without_tax}}
             {{lang 'products.price_with_tax' tax_label=price.tax_label}}
         {{/if}}
     </div>
 {{/if}}
-
 {{#if price.without_tax}}
```

Comment:
wondering if we should use an `{{#else}}` here - can (or should) both `price.with_tax` and `price.without_tax` every be falsey simultaneously?